### PR TITLE
Rename Bundle metadata type to BundleMetadata

### PR DESCRIPTION
**Pull request by schmidt-sebastian**
Rename to match API names in the BundleCache.

### CODE REVIEW

File: packages/firestore/src/local/bundle_cache.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export interface BundleCache {
   getBundleMetadata(
     transaction: PersistenceTransaction,
     bundleId: string
-  ): PersistencePromise<Bundle | undefined>;
+  ): PersistencePromise<BundleMetadata | undefined>;
```
Comment (thebrianchen) [inline packages/firestore/src/local/bundle_cache.ts:38]: s/Bundle/BundleMetadata in L32

File: packages/firestore/src/local/local_serializer.ts [hunk 2]
```diff
@@ -307,7 +307,7 @@ function isDocumentQuery(dbQuery: DbQuery): dbQuery is PublicDocumentsTarget {
 }
 
 /** Encodes a DbBundle to a Bundle. */
```
Comment (thebrianchen) [inline packages/firestore/src/local/local_serializer.ts:309]: Rename Bundle here as well?

File: packages/firestore/src/local/local_serializer.ts [hunk 3]
```diff
@@ -373,7 +373,9 @@ export function fromProtoNamedQuery(namedQuery: ProtoNamedQuery): NamedQuery {
 }
 
 /** Encodes a BundleMetadata proto object to a Bundle model object. */
-export function fromBundleMetadata(metadata: ProtoBundleMetadata): Bundle {
+export function fromBundleMetadata(
```
Comment (thebrianchen) [inline packages/firestore/src/local/local_serializer.ts:376]: Optional jsdoc rename here as well.

**Review by thebrianchen — APPROVED** [review]

File: packages/firestore/src/local/local_serializer.ts (hunk 2 above)
Comment (schmidt-sebastian) [reply packages/firestore/src/local/local_serializer.ts:309]: Done

File: packages/firestore/src/local/local_serializer.ts (hunk 3 above)
Comment (schmidt-sebastian) [reply packages/firestore/src/local/local_serializer.ts:376]: Done

File: packages/firestore/src/local/bundle_cache.ts (hunk 1 above)
Comment (schmidt-sebastian) [reply packages/firestore/src/local/bundle_cache.ts:38]: Done

**Pull request merged**